tools/hdr/local_tone_map.cc

```diff
@@ -284,12 +285,14 @@ Status ProcessFrame(CodecInOut* image, float preserve_saturation,
 
   JXL_RETURN_IF_ERROR(GamutMap(image, preserve_saturation, pool));
 
-  ColorEncoding rec2020_srgb = linear_rec2020;
-  rec2020_srgb.Tf().SetTransferFunction(TransferFunction::kSRGB);
-  JXL_RETURN_IF_ERROR(rec2020_srgb.CreateICC());
+  ColorEncoding displayP3;
+  JXL_RETURN_IF_ERROR(displayP3.SetWhitePointType(WhitePoint::kD65));
+  JXL_RETURN_IF_ERROR(displayP3.SetPrimariesType(Primaries::kP3));
+  displayP3.Tf().SetTransferFunction(TransferFunction::kSRGB);
+  JXL_RETURN_IF_ERROR(displayP3.CreateICC());
   JXL_RETURN_IF_ERROR(
-      image->Main().TransformTo(rec2020_srgb, *JxlGetDefaultCms(), pool));
-  image->metadata.m.color_encoding = rec2020_srgb;
+      image->Main().TransformTo(displayP3, *JxlGetDefaultCms(), pool));
```

Comment:
In a sense, there are now two successive gamut reductions (tone mapping result ->(`GamutMap(preserve_saturation)`)->Rec. 2020->(`TransformTo(displayP3)`)->P3), but maybe that’s not too much of a problem.

(This `TransformTo` was previously really just for the sRGB TRC.)